cmd/identity/revocations.go

```diff
@@ -53,12 +53,7 @@ func cmdRevocations(cmd *cobra.Command, args []string) error {
 
 	revErrs := new(errs.Group)
 	for _, rev := range revs {
-		// TODO: figure out why this doesn't base64 encode []byte fields!
-		//revJSON, err := json.MarshalIndent(rev.CertHash, "", "\t")
-		//if err != nil {
-		//	revErrs.Add(err)
-		//}
-		fmt.Printf("certificate hash: %s\n", base64.StdEncoding.EncodeToString(rev.CertHash))
+		fmt.Printf("certificate public key hash: %s\n", base64.StdEncoding.EncodeToString(rev.KeyHash))
```

Comment:
Do we still want these print lines ?

Reply:
yes, this command prints revocation information,